<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessELO</class>
 <widget class="QMainWindow" name="ChessELO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessELO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="separationLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Separation:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>ELO -</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Calibri&quot;;
color: rgb(0, 170, 255)</string>
    </property>
    <property name="text">
     <string>Transitive Checkmates</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ELOLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="descripLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Find the number of players separating you from the highest ELO player.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Name -</string>
    </property>
   </widget>
   <widget class="QLabel" name="separationLabel_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="highestEloLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Highest ELO Player:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>180</y>
      <width>437</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="searchPlayerLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Search Player Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Enter">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
